src/components/AddRepositoryDetails/AddRepo.tsx

```diff
@@ -37,36 +47,45 @@ interface Orgs {
     url: string;
 }
 
-interface ComputeService {
-    id: string;
-    name: string;
-}
 
   
 export default function AddRepo() {
     // UseState
     const dashboardController = useDashboardController();
     const [selectValue, setSelectValue] = useState<string>("");
+    const [singleChecked, setSingleChecked] = useState<boolean>(false);
+    const [singleCheckedData, setSingleCheckedData] = useState<RepoTable>({"id": "", "name": "", "link": "", "org": ""});
+    const [selectComputeService, setSelectComputeService] = useState<string>("");
+
     const [octo, setOcto] = useState<Octokit>(dashboardController.octokit);
```

Comment:
this value will never change, so you can simplify this to:
`const octo = dashboardController.octokit;`

Reply:
Updated